drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/events/EnableTestToolsEvent.java

```diff
@@ -37,9 +39,9 @@
     private final boolean notEqualsSearch;
 
     /**
-     * The string to <b>eventually</b> use to select the property in the test tools  panel
+     * The <code>List</code> to <b>eventually</b> use to select the property in the test tools  panel
      */
-    private final String propertyName;
+    private final List<String> propertyNameElements;
```

Comment:
Just an idea, but would it make sense to wrap `List<String>` to a `PropertyNameElements` class, so that it is clear what the list is about and to perhaps move some actions done against the list into the `PropertyNameElements` class?

Reply:
Same goes for `List<String> headerCellElements`, not sure if the two can share a class.

Reply:
@Rikkola 
Actually, in general I like the idea of having a more semantically-defined class. But for this specific situation my thoughts were
1) there is no fixed number of elements - so I should have anyway a list behind the scene
2) creating a specific class would require anyway some "effort" (code to write/maintain, small as it may be), that for this specific situation probably won't be paid off

Anyway, if for some reason there would be the needs/option to have something more specific instead of a List, I'll do

Reply:
Sure. I think here your the author so you decide since you are the one that has the better understanding on it.